receiver/stats.go

```diff
@@ -0,0 +1,155 @@
+/*
+ * skogul, stats receiver
+ *
+ * Copyright (c) 2019-2020 Telenor Norge AS
```

Comment:
Should say just Copyright (c) 2021

Reply:
Damnit. I remember changing this more than once during my copy/paste endeavours. I'll fix.